paymentsheet/src/test/java/com/stripe/android/paymentsheet/VerticalModePaymentSheetActivityTest.kt

```diff
@@ -446,6 +446,7 @@ internal class VerticalModePaymentSheetActivityTest {
         // Verify that the error message appears
         formPage.assertErrorExists("American Express is not accepted")
         verticalModePage.assertPrimaryButton(isNotEnabled())
+        formPage.fillCardNumber("")
```

Comment:
Why is this necessary?

Reply:
I notice some flaks occasionally, where when entering an Amex the CVC field changing from CVC to CVV, then "pasting" in an entire card doesn't update it back to CVC. Seems like an unrelated bug.

The solution here is probably to just remove CVV usage for Amex, we did this on iOS and web no longer uses CVV.